src/Text/Pretty/Simple/Internal/Printer.hs

```diff
@@ -291,27 +293,42 @@ data Annotation
   | Quote
   | String
   | Num
+  | CustomAnn Style
 
--- | Apply various transformations to clean up the 'Expr's.
-preprocess :: OutputOptions -> [Expr] -> [Expr]
-preprocess opts = map processExpr . removeEmptyOthers
+--TODO split up (with each step expressed with makePostProcessor?) so we can pick and choose
+defaultPostProcess :: StringOutputStyle -> [Expr] -> [Expr]
+defaultPostProcess stringStyle = map processExpr . removeEmptyOthers
```

Comment:
This has effectively changed name from 'preprocess' to 'postprocess'... Really it's just a matter of perspective